zone/client.cpp

```diff
@@ -8340,6 +8340,11 @@ void Client::SetThirst(int32 in_thirst)
 	safe_delete(outapp);
 }
 
+void Client::SetIntoxication(int32 in_intoxication)
+{
+	m_pp.intoxication = std::min(200,std::max(0,in_intoxication));
```

Comment:
`EQ::Clamp(in_intoxication, 0, 200)` would work here. Just have to include `common/data_verification.h`.

Reply:
Added